docs/md/mainpage.md

```diff
@@ -0,0 +1,42 @@
+\mainpage Introduction
+
+\tableofcontents
+
+\section ckpt_what_is What is VT TV?
+VT TV is ...
+
+\section License
+
+@m_class{m-note m-dim}
+
+@parblock
+Copyright 2021-2024 National Technology & Engineering Solutions of Sandia, LLC
```

Comment:
@maxime-bfsquall @ppebay @lifflander The LICENSE file at the top of the repo says `Copright 2019`. Which one is correct?

Reply:
We should make sure that this verification is made common to repositories @maxime-bfsquall @cwschilly